src/main/java/school/lemon/changerequest/java/io/hw1/IFileManagerImp.java

```diff
@@ -0,0 +1,625 @@
+
+        if (file.isDirectory()) {
+            size = 0;
+        } else {
+            size = file.length();
+        }
+
+        SimpleDateFormat sdf = new SimpleDateFormat("dd.MM.yyyy HH:mm:ss");
+        String str1 = sdf.format(file.lastModified());
+        if (file.isDirectory()) str2 = file.getName() + "/";
+        else str2 = file.getName();
+
+        return String.format("%-20s %15s %s %n", str1 + divider, size + divider, str2);
+    }
+
+    /**
+     * Create new file within current directory with given name
+     *
+     * @param name name of new file
+     * @throws FileAlreadyExistException if file or directory with given name already exists
+     * @throws FileManagerException      if any other problem happened
+     */
+    @Override
+    public void createFile(String name) throws FileAlreadyExistException, FileManagerException {
+        File file = new File(name);
+
+        try {
+            if (file.isDirectory()) {
+                throw new IOException();
+            }
+            boolean fcnf = file.createNewFile();
+            if (fcnf) {
+                System.out.println("File has been created successfully");
+            } else {
+                throw new FileAlreadyExistException(file);
+            }
+        } catch (FileAlreadyExistException n) {
+            System.out.println(n.getMessage());
+        } catch (FileManagerException n) {
+            System.out.println(n.getMessage());
+        } catch (IOException e) {
+            System.out.println("Not File");
+        }
+    }
+
+
+    @Override
+    public void createDirectory(String name) throws DirectoryAlreadyExistException, FileManagerException {
+        File file = new File(name);
+        try {
+            if (file.isDirectory()) {
+                throw new IOException();
+            }
+            boolean fcnf = file.mkdir();
+            if (fcnf) {
+                System.out.println("Directory has been created successfully");
+            } else {
+                throw new DirectoryAlreadyExistException( file);
+            }
+        } catch (DirectoryAlreadyExistException n) {
+            System.out.println(n.getMessage());
+        } catch (FileManagerException n) {
+            System.out.println(n.getMessage());
+        } catch (IOException e) {
+            System.out.println("Not Directory");
+        }
+
+    }
+
+    @Override
+    public void changeDirectory(String name) throws IllegalDirectoryException {
+
+        try {
+            Paths.get(name);
+            setPath(name);
+            System.out.println("Changed Directory");
+        } catch (IllegalDirectoryException ide) {
+            System.out.println(ide.getMessage());
+        }
+    }
+
+    @Override
+    public boolean delete(String name) throws FileNotFoundException {
+        try
+        {
+            File file = new File(name);
+            if(file.exists())
+            {
+                file.delete();
+                return true;
```

Comment:
Are you sure that file was actually deleted? Please read documentation for `file.delete` method.